benchmark/compute_metrics.py

```diff
@@ -237,7 +239,7 @@ def compute_metric_set(out, gt, metric_set):
         ])
     elif metric_set.startswith('full_'):
         aligned = align_to_gt(out, gt, -1, **metric_set_to_alignment_params(metric_set))
```

Comment:
First call to `get_overlapping_xyz_parts_on_same_time_grid()` happens here. Another way to fix the issue would be to not recompute the interpolation below, but it seemed like it would be messier.